Orange/widgets/regression/owlinearregression.py

```diff
@@ -124,6 +157,9 @@ def commit(self):
             learner = RidgeRegressionLearner(alpha=alpha, **args)
         elif self.reg_type == OWLinearRegression.Lasso:
             learner = LassoRegressionLearner(alpha=alpha, **args)
+        elif self.reg_type == OWLinearRegression.Elastic:
```

Comment:
This if-elif construct is missing an else statement and is potentially problematic.
I just tested this pull request and tried the Elastic option. Then on the master branch, Orange crashed because the learner was not set (stored widget settings contained reg_type=3).
The problem is of course because of incorrect settings, but it would not fail so badly if this had a proper else clause.